src/logic/safe/utils/shouldSafeStoreBeUpdated.ts

```diff
@@ -0,0 +1,108 @@
+import { SafeRecordProps } from 'src/logic/safe/store/models/safe'
```

Comment:
do you think is it possible to use https://lodash.com/docs/4.17.15#isEqual instead?

Reply:
yes, with the comment of Fer I thought it's better to use that instead, doing it, will ping you once ready

Reply:
It's added now